sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -232,14 +233,15 @@ public partial class TableSasBuilder
         public string RowKeyStart { get { throw null; } set { } }
         public System.DateTimeOffset StartsOn { get { throw null; } set { } }
         public string TableName { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public string Version { get { throw null; } set { } }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
         public void SetPermissions(Azure.Data.Tables.Sas.TableSasPermissions permissions) { }
         public void SetPermissions(string rawPermissions) { }
-        public Azure.Data.Tables.Sas.TableSasQueryParameters ToSasQueryParameters(Azure.Data.Tables.TableSharedKeyCredential sharedKeyCredential) { throw null; }
+        public string Sign(Azure.Data.Tables.TableSharedKeyCredential sharedKeyCredential) { throw null; }
```

Comment:
I don't think we should delete the old one because you'll still need something that integrates with your TableUriBuilder - if you have a TableUriBuilder.  Otherwise, maybe we don't need TableSasQueryParameters either?

Reply:
No, I don't have a TableUriBuilder - the baseline Uri is established when constructing the TableServiceClient. The only two scenarios I've encountered so far are to use the [base table endpoint](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/tests/TableServiceClientTests.cs#L20) or a [sas generated uri](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs#L49-L51). 

You are probably right that TableSasQueryParameters could be consolidated to the TableSasBuilder. Do you see some benefit in doing that?

Reply:
The [`*UriBuilder`s](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobUriBuilder.cs) are more for customers than us.  Some of these URIs end up being pretty complicated and it's helpful for folks to be able to slice and dice them.  I think you _may_ want to add one at some point and I think some of your code paths are designed to support one (like the .ctor that removes all the SAS params from a dictionary).

Let's leave it here for this PR and keep the `ToSasQueryParameters` method you can just call into from `Sign`.  We can decided to either add `TableUriBuilder` or remove `TableSasQueryParameters` in the future.